src/main/java/com/google/sps/servlets/ImagesServlet.java

```diff
@@ -1,35 +1,84 @@
-package main.java.com.google.sps.servlets;
+package com.google.sps.servlets;
 
+//getting the images.java class...
+import com.google.sps.data.Images;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
```

Comment:
Can do this later but keeping imports organized alphabetically is something we like to do. The reason is, when the compiler complains about "Can't find com.xyz.abc.jkl.SomeClass`, you can easily look through a hundred imports and see if you forgot it.